tools/plugins/com.liferay.ide.portlet.core/src/com/liferay/ide/portlet/core/operation/NewPortletClassDataModelProvider.java

```diff
@@ -1024,7 +1024,7 @@ else if( CREATE_RESOURCE_BUNDLE_FILE_PATH.equals( propertyName ) )
             }
             else
             {
-                return PortletCore.createWarningStatus( Msgs.resourceBundleFilePathEndWithProperties );
```

Comment:
here is the sdk build-lang command, they did hardcode in it
<target name="build-lang">
		<if>
			<available file="docroot/WEB-INF/src/content" />
			<then>
				<antcall target="build-lang-cmd">
					<param name="lang.dir" value="docroot/WEB-INF/src/content" />
					<param name="lang.file" value="Language" />
					<param name="lang.translate" value="true" />
				</antcall>
			</then>
		</if>

		<if>
			<available file="src/content" />
			<then>
				<antcall target="build-lang-cmd">
					<param name="lang.dir" value="src/content" />
					<param name="lang.file" value="Language" />
					<param name="lang.translate" value="true" />
				</antcall>
			</then>
		</if>
	</target>